geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSetCommandsFunctionExecutor.java

```diff
@@ -43,4 +43,9 @@ public long zadd(RedisKey key, List<byte[]> scoresAndMembersToAdd, ZAddOptions o
   public byte[] zscore(RedisKey key, byte[] member) {
     return stripedExecute(key, () -> getRedisSortedSet(key, true).zscore(member));
   }
+
+  @Override
+  public long zcard(RedisKey key) {
+    return stripedExecute(key, () -> getRedisSortedSet(key, true).zcard());
```

Comment:
Could we add a test to `AbstractHitsMissesIntegrationTest` verifying that stats should be updated for `ZCARD`?